UndertaleModLib/UndertaleData.cs

```diff
@@ -467,15 +467,31 @@ private bool TestGMS1Version(uint stableBuild, uint betaBuild, bool allowGMS2 =
         /// <param name="minor">The minor version.</param>
         /// <param name="release">The release version.</param>
         /// <param name="build">The build version.</param>
-        public void SetGMS2Version(uint major, uint minor = 0, uint release = 0, uint build = 0)
+        /// <param name="isLTS">If included, alter the data branch between LTS and non-LTS.</param>
+        public void SetGMS2Version(uint major, uint minor = 0, uint release = 0, uint build = 0, bool? isLTS = null)
         {
             if (major != 2 && major != 2022 && major != 2023 && major != 2024)
                 throw new NotSupportedException("Attempted to set a version of GameMaker " + major + " using SetGMS2Version");
 
-            GeneralInfo.Major = major;
+            GeneralInfo.Major = (uint)major;
             GeneralInfo.Minor = minor;
             GeneralInfo.Release = release;
             GeneralInfo.Build = build;
+
+            if (isLTS is not null)
+            {
+                SetLTS((bool)isLTS);
+            }
+        }
+
+        /// <summary>
+        /// Sets the branch type in GeneralInfo to the appropriate LTS or non-LTS version based on 
```

Comment:
Based? Based on what? (sorry)

Reply:
based on the branch thats inside the data.win